cmake/public/ComputeLibrary.cmake

```diff
@@ -0,0 +1,41 @@
+# Build with Compute Library backend for the Arm architecture
+# Note: Compute Library is available from: https://github.com/ARM-software/ComputeLibrary
+#   and must be built separately. The location of the Compute Library build
+#   must be set with the env var ACL_ROOT_DIR. This path will be checked later
+#   as part of FindACL.cmake in oneDNN.
+
+IF(NOT CPU_AARCH64)
```

Comment:
Please use lowercase spelling of CMake commands (for consistency with rest of files in the folder)

Reply:
Done.